cdap-client/src/main/java/co/cask/cdap/client/ApplicationClient.java

```diff
@@ -165,6 +165,7 @@ public ApplicationClient(ClientConfig config) {
    * @throws IOException if a network error occurred
    * @throws UnauthenticatedException if the request is not authorized successfully in the gateway server
    */
+  @Deprecated
   public ApplicationDetail get(Id.Application appId)
     throws ApplicationNotFoundException, IOException, UnauthenticatedException, UnauthorizedException {
```

Comment:
Theoretically everything going to the versioned API with a default version should return the same result as non-versioned API. I'll just delegate this to the new API.